signals.go

```diff
@@ -19,15 +19,14 @@ import (
 //
 // The value is true if receiving the signal should be fatal.
 var handledSignalsMap = map[syscall.Signal]bool{
-	syscall.SIGABRT:   true,
-	syscall.SIGBUS:    true,
-	syscall.SIGILL:    true,
-	syscall.SIGQUIT:   true,
-	syscall.SIGSEGV:   true,
-	syscall.SIGSTKFLT: true,
```

Comment:
@raravena80 could you please add a commit message explaining a bit what this patch is doing, and particularly the removal of `SIGSTKFLT`.
Maybe splitting this into two commits might help clarifying that too.

Reply:
BTW, are you planning to implement what you mentioned in the issue? I mean having this `signals.go ` as a common part between different OSes, and define `signals_linux.go` and `signals_darwin.go` for specific ones ?

Reply:
@raravena80 Would make sense to split this in two commits, as the changes are unrelated. Other than that 
LGTM

Reply:
if you have the file name extentions (_$GOARCH), then you don't need the in-file tags, do you?

Reply:
Oh interesting, looking here: https://golang.org/pkg/go/build/, you're right. But it cannot hurt to put the build flag on top of each file, just as an enforcement action.

Reply:
Right, you don't need it but like @sboeuf mentioned it may be good to keep for informational purposes :)

Reply:
Regarding build flag vs. filename, I'm not sure which one "trumps" the other. I'd rather we just use the filename as that's the convention we're using and we don't want to have to maintain two things for the same purpose ;)

Reply:
sounds good. I just removed the `+build` header.